source/manual/aws-cli-access.html.md

```diff
@@ -0,0 +1,57 @@
+---
+owner_slack: "#govuk-2ndline"
+title: Use AWS on the command line
+section: AWS accounts
+layout: manual_layout
+parent: "/manual.html"
+last_reviewed_on: 2018-10-10
+review_in: 3 months
+---
+
+💡 Before you can access something in AWS, you [need to set up your AWS account](/manual/set-up-aws-account.html).
+
+---
+
+## 1. Create `~/.aws/config`
+
+You will need 2 pieces of information:
+
+1. `role_arn`: Find the ["Role ARN" in the docs][secret-docs]
+1. Your `mfa_serial`: the "MFA ARN" assigned to the MFA device in your account
+
+Create `~/.aws/config`:
+
+```ini
+[profile govuk-<environment>]
+source_profile = gds
+role_arn = <Role ARN>
+mfa_serial = <MFA ARN>
+
+[profile gds]
+region = eu-west-1
+```
+
+## 2. Create `~/.aws/credentials`
+
+First, create a pair of [access keys][access-keys].
```

Comment:
Todo: explain where to generate & find these.

Reply:
Do we explain this somewhere already? If that's the case, we can simply add a link to that explanation.

Reply:
I don't think we do? Have you seen it somewhere?

Reply:
No, I haven't seen it. I think I followed AWS own guide.